packages/grafana-ui/src/themes/ThemeContext.test.tsx

```diff
@@ -1,23 +1,52 @@
-import React from 'react';
 import { config } from '@grafana/runtime';
+import { renderHook } from '@testing-library/react-hooks';
 import { css } from 'emotion';
 import { mount } from 'enzyme';
-import { useStyles } from './ThemeContext';
+import React from 'react';
```

Comment:
Maybe this is just my ocd, but why are you moving this import?

Reply:
I rarely manage imports manually these days, must've been vscode. :)
But you're absolutely right - it's ugly and I fixed it.